packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.test.js

```diff
@@ -1224,5 +1348,59 @@ describe('InputDateTimePicker', () => {
 				},
 			],
 		);
+
+		cases(
+			'should NOT apply an "invalid placeholder" on input AND override the regular one when InvalidDate value is internal, ie: wrong input value to keep visible',
```

Comment:
This seems to be 2 different tests as you execute different branches using `if`. Please split it into 2 simple tests

Reply:
✔